.gitignore

```diff
@@ -297,3 +297,9 @@ functions-extensions/
 .vscode/
 .ionide/
 /src/WebJobs.Extensions.DurableTask/Microsoft.Azure.WebJobs.Extensions.DurableTask.xml
+
+# E2E Tests build output
+/test/DotnetIsolatedE2ETests/Azure.Functions.Cli/*
```

Comment:
Is this where we put the CLI or does it go into a temp directory (in which case we don't need this .gitignore line)?

Reply:
Removed this and the last line from .gitignore, not needed now